src/test/java/de/tum/in/www1/artemis/ModelingAssessmentIntegrationTest.java

```diff
@@ -92,13 +96,21 @@ public void initTestCase() throws Exception {
         activityExercise = (ModelingExercise) exerciseRepo.findAll().get(1);
         objectExercise = (ModelingExercise) exerciseRepo.findAll().get(2);
         useCaseExercise = (ModelingExercise) exerciseRepo.findAll().get(3);
+        validModel = database.loadFileFromResources("test-data/model-submission/model.54727.json");
     }
 
     @AfterEach
     public void tearDown() {
         database.resetDatabase();
     }
 
+    @Test
+    @WithMockUser(username = "tutor1", roles = "TA")
+    public void testGetPartialAssessment_notFound() throws Exception {
```

Comment:
@krusche Actually, the REST call for `getPartialAssessment` is currently not used. Is there any plan to work on the TODOs listed for this method or should I remove the unused code regarding this REST call from the client/server code?
https://github.com/ls1intum/Artemis/blob/b183606d026b981afb7a50f99d3e6f109fc5b93e/src/main/java/de/tum/in/www1/artemis/web/rest/ModelingAssessmentResource.java#L83

Reply:
please remove it in the server and in the client if we don't use it at the moment